aws/Dockerfile

```diff
@@ -6,7 +6,8 @@ ARG CI_HELPER_VERSION
 WORKDIR /root
 RUN echo "Install AWS" && \
     apk add -q --no-cache bash ca-certificates curl git make openssh-client python py-pip rsync zip && \
-    pip -q install awscli boto3 PyYAML && \
+    pip -q install pip==18 \
```

Comment:
Should add arguments to configure it in `.travis.yml`

Reply:
Done